data/factions/077_Tolan_Kingdom.lua

```diff
@@ -0,0 +1,27 @@
+-- Copyright © 2008-2024 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local f = Faction:new('Tolan Kingdom')
+	:description_short('Tolan Kingdom')
+	:description('The Tolan Kingdom, famous (or infamous, depending on who you ask) for repelling Haber invasion forces 500 years ago.')
+	:homeworld(2,-9,-19,2,4) -- edge of Habers sphere of influence
+	:foundingDate(2337) -- as per timeline
+	:expansionRate(0) -- a single system only, but quite populous
```

Comment:
Per current lore / IRC discussion (CC @bszlrd) the Tolan Kingdom is a small, insular faction spanning a few systems. Either the expansionRate variable should be a small non-zero value, or a few specific systems assigned to the faction (probably surrounding Tolan but not intruding too far into Haber space).

Reply:
OK. 0.01 gives the Tolans a dozen or so system.